src/main/java/mate/academy/spring/controller/UserControllerImpl.java

```diff
@@ -0,0 +1,50 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.UserResponseDto;
+import mate.academy.spring.service.UserService;
+import mate.academy.spring.service.mapper.UserMapper;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/users")
+public class UserControllerImpl {
+    private final UserService userService;
+    private final UserMapper userMapper;
+
+    public UserControllerImpl(UserService userService, UserMapper userMapper) {
+        this.userService = userService;
+        this.userMapper = userMapper;
+    }
+
+    @GetMapping("/{id}")
+    public UserResponseDto get(@PathVariable Long id) {
+        return userMapper.parse(userService.get(id));
```

Comment:
```suggestion
        return userMapper.parseToDto(userService.get(id));
```
we'll have response and request DTOs, so it's better to specify which way you map the object